app/models/ext_management_system.rb

```diff
@@ -316,8 +311,12 @@ def self.ems_infra_discovery_types
     EMS_INFRA_DISCOVERY_TYPES.values
   end
 
+  def self.register_cloud_discovery_type(type_hash)
+    ems_cloud_discovery_types.merge!(type_hash)
+  end
+
   def self.ems_cloud_discovery_types
-    EMS_CLOUD_DISCOVERY_TYPES
+    @ems_cloud_discovery_types ||= {'azure'  => 'azure'}
```

Comment:
I thought so, but actually I think I can just do this in this PR, should be a one liner